scripts/ddd/ddd

```diff
@@ -95,7 +95,11 @@ restore_mapping() {
 
 	# If we have stored USB_MAP, reload it
 	if [ -f ${DDD_MAP} ]; then
-		source ${DDD_MAP}
+        source ${DDD_MAP}
```

Comment:
I agree! And I get Bash being your favroite - it's been really growing on me the past few months